integration/helpers/helpers.go

```diff
@@ -62,19 +62,21 @@ import (
 
 // CommandOptions controls how the SSH command is built.
 type CommandOptions struct {
-	ForwardAgent bool
-	ForcePTY     bool
-	ControlPath  string
-	SocketPath   string
-	ProxyPort    string
-	NodePort     string
-	Command      string
+	ForwardAgent  bool
+	ForcePTY      bool
+	X11Forwarding bool
+	ControlPath   string
+	SocketPath    string
+	ProxyPort     string
+	NodePort      string
+	Command       string
 }
 
 // ExternalSSHCommand runs an external SSH command (if an external ssh binary
 // exists) with the passed in parameters.
 func ExternalSSHCommand(o CommandOptions) (*exec.Cmd, error) {
 	var execArgs []string
+	execArgs = append(execArgs, "-v")
```

Comment:
Why do you have to put it in verbose mode?

Reply:
Removed, I was using it for debugging the test.